deployment/helm/synopsys-alert/templates/_helpers.tpl

```diff
@@ -32,3 +32,31 @@ Environs for Alert Config Map
 {{ $key }}: {{ $value | quote }}
 {{- end }}
 {{- end -}}
+
+{{/*
+Enable SSL for External Postgres
+*/}}
+{{- define "enablePostgresSSL" -}}
+{{- if and (eq .Values.postgres.isExternal true) (eq .Values.postgres.ssl true) -}}
+{{- if (eq .Values.postgres.sslConfig.sslUseFiles true) -}}
+ALERT_DB_SSL_MODE: "verify-ca"
+{{- else -}}
+ALERT_DB_SSL_MODE: "require"
+{{- end -}}
+{{- else -}}
+ALERT_DB_SSL_MODE: "disable"
+{{- end -}}
+{{- end -}}
+
+{{/*
+Use SSL Files for External Postgres
+*/}}
+{{- define "postgresSSLFiles" -}}
+{{- if and (eq .Values.postgres.isExternal true) (eq .Values.postgres.ssl true) -}}
+{{- if (eq .Values.postgres.sslConfig.sslUseFiles true) -}}
+ALERT_DB_SSL_KEY_PATH: {{ .Values.postgres.sslConfig.sslKeyFilePath }}
```

Comment:
If we are using secrets and we probably should just require secrets be created for those files, then do we need the file path options in the templates?  We look for the secrets in the templates below.  When would the file paths be used over the secret use case?

Reply:
I kept the non-secret use case as we do something similar with DB credentials. This could have been for backwards compatibility but I wasn't sure so I added the ability to add them directly or to use secrets. If it was only for backwards compatibility, I can certainly use secrets or the other.

Reply:
The DB credentials are also strings.  We simply need the text in the case of credentials.  Here we need files stored.  Secrets are going to be stored on the system as files in the secrets directory.  For Helm I would prefer to just support secrets to have less to maintain and support.  These are SSL related files they should be treated like passwords.  Unless anyone else strongly objects.